README.rst

```diff
@@ -17,25 +17,63 @@ The widget takes the following parameters:
  - override: drops all global config and the base query if a list is passed to the widget. All types need to be added to be selectable.
 
 
-**IMPORTANT NOTE:**
-Currently this widget drops the SourceBinder concept, which has huge impact on the usability.
+ContextSourceBinder
+-------------------
+
+With a `RelationeChoice` or `RelationList` of `RelationChoice` a source can be configured along with the field.
+The ContextSourceBinder makes sure, the right content will be referenced.
+
+The default filter method, therefore only checks for the correct type:
```

Comment:
I would try to rephrase that. Maybe:

By default, the source binder only checks for a valid portal_type when selecting content.
The `default_filter` implementation therefore looks like this: